src/main/java/com/musala/sdcs/devices/PlayControllType.java

```diff
@@ -0,0 +1,7 @@
+package com.musala.sdcs.devices;
+
+public enum PlayControllType {
```

Comment:
Please rename the class/java file to `PlayControlType` (single L) :) Add JavaDoc.